test/connectors/ReportDeadlinesConnectorSpec.scala

```diff
@@ -50,7 +54,7 @@ class ReportDeadlinesConnectorSpec extends TestSupport {
       }
 
       "called for fulfilled obligations" in new Setup {
-        val date: LocalDate = LocalDate.now()
+        val date: LocalDate = new DateService().getCurrentDate
```

Comment:
you already have dateService in scope, 
so you can just call it =>
` val date: LocalDate = dateService.getCurrentDate`